docs: accts, idp, and disclosure component (#3102)
interledger/rafiki: packages/documentation/src/partials/auth-variables.mdx

| `AUTH_DATABASE_URL`      | `auth.postgresql.host`,<br />`auth.postgresql.port`,<br />`auth.postgresql.username`,<br />`auth.postgresql.database`,<br />`auth.postgresql.password` | `postgresql://postgres:password@localhost:5432/auth_development` | The URL of the Postgres database storing your Open Payments grant data. For Helm, these components are provided individually.       |
| `AUTH_SERVER_URL`        | `auth.server.domain`                                                                                                                                   | _undefined_                                                      | The public endpoint for your Rafiki instance’s public Open Payments routes.                                                         |
| `COOKIE_KEY`             | `auth.cookieKey`                                                                                                                                       | _undefined_                                                      | The <LinkOut href="https://koajs.com/#app-keys-">koa KeyGrip key</LinkOut> that is used to sign cookies for an interaction session. |

> | `IDENTITY_SERVER_URL`    | `auth.identityServer.domain`                                                                                                                           | _undefined_                                                      | The URL of your IdP's server, used by your authorization server to inform an Open Payments client of where to redirect the end-user to complete the interaction and authorize a grant. |

Reviewer: ```suggestion
| `IDENTITY_SERVER_URL`    | `auth.identityServer.domain`                                                                                                                           | _undefined_                                                      | The URL of your IdP's server, used by your authorization server to inform an Open Payments client of where to redirect the end-user to start interactions. |
```
